hack/kind-multi-node-cluster-without-registry.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
I think it makes sense to have one single `kind-create.sh` script that accepts parameters.
Also, having the configurations as separate files (and passing them as a parameter) could be a cleaner approach.
I'll file a separate issue, as I don't want this to hold